atlp-rwanda/hackers-ec-Fe
# Landing page

#### What does this PR do?
- it contains landing page sections

#### How should this be manually tested?
- crone the repo 
- navigate to my pr repo
- pull it from aorigin
- run ```npm install```
- run ```nupm run dev```
- check the pages
